lib/modules/platform/gitea/index.ts

```diff
@@ -319,12 +319,25 @@ const platform: Platform = {
   async getRepos(config?: AutodiscoverConfig): Promise<string[]> {
     logger.debug('Auto-discovering Gitea repositories');
     try {
-      if (!config?.topics) {
+      if (config?.topics) {
+        logger.debug('Auto-discovering Gitea repositories by topics');
```

Comment:
@rarkins should we log topics here? just in case force easier trouble shooting?

Reply:
@viceice let me know if we need to create additional logging

Reply:
```suggestion
        logger.debug({ topics: config.topics }, 'Auto-discovering by topics');
```